### PR TITLE
Feature/vlan filter engine

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Vlan filters based off the filter engine depends on feature/filter-engine
## Impacts

The vlan filters
## NEWS file entries

Enhancements
++++++++++++
- Move to the vlan filters to use the feature/filter-engine
## Delete branch after merge

NO

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
Review me

File: addons/pfconfig/comparator/dumper.pl [hunk 1]
```diff
@@ -65,10 +65,10 @@ =head1 DESCRIPTION
 }
 
 {
-  use pf::vlan::filter;
+  use pf::access_filter::vlan;
 
   my @variables = ('%ConfigVlanFilters');
-  $configs{'pf::vlan::filter'} = dump_module("pf::vlan::filter", @variables);
+  $configs{'pf::access_filter::vlan'} = dump_module("pf::vlan::filter", @variables);
```
Comment (fdurand) [inline addons/pfconfig/comparator/dumper.pl]: pf::vlan::filter doesn't exist anymore

File: lib/pf/access_filter.pm [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+=head1 DESCRIPTION
+
+pf::access_filter deny, rewrite role based on rules.
+
+=cut
+
+use strict;
+use warnings;
+
+use Log::Log4perl;
```
Comment (fdurand) [inline lib/pf/access_filter.pm]: use pf::log;

**Comment by jrouzierinverse** [comment]
Please review

**Comment by julsemaan** [comment]
Requires some upgrade notes

File: lib/pf/access_filter/vlan.pm [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+=head2 dispatchAction
+
+Return the reference to the function that call the api.
+
+=cut
+
+sub dispatchAction {
+    my ($self, $rule, $args);
```
Comment (fdurand) [inline lib/pf/access_filter/vlan.pm]: = @_;

**Pull request merged**